README.md

```diff
@@ -92,17 +92,17 @@ The `alfresco-docker-base-java` image can be generated in multiple flavors by mi
 
 #### Java
 
-Either Java 8 (supported up to 5.2 and 6.0) or Java 11 can used used to build images using the `JAVA_MAJOR` build argument.
+Either Java 11 or Java 17 can used used to build images using the `JAVA_MAJOR` build argument.
```

Comment:
we can fix the existing error "can used used" to "can be used"

Reply:
> we can fix the existing error "can used used" to "can be used"

Fixed it in new commit